### ISSUE TITLE
Feature: Link to Github Page or Documentation

### ISSUE THREAD

**Issue by raviwarrier** [hoarder-app/hoarder#533]
You could make this:

![image](https://github.com/user-attachments/assets/fc69d618-dbb8-46c8-86e9-313c769449ca)

a link to the github page. just a convenient way to reach here.

if you want to extend it, it could be a menu with these options: Github Page, Releases, Documentation/Help.

**Comment by kamtschatka** [comment]
yes, there are already mockups for this feature in discord. (link to github)

**Comment by InventorB** [comment]
@kamtschatka  was this ever implemented or is this still open to work on?

**Comment by raviwarrier** [comment]
> [@kamtschatka](https://github.com/kamtschatka) was this ever implemented or is this still open to work on?

Something's implemented. Clicking on the version number, opens a modal window with the changelog. And it has a link "View on Github". 

And documentation is in the menu, so I guess it covers all the things I had suggested.
